### DISCUSSION TITLE
Kernel used in Brunch r94 stable 20211121

### DISCUSSION THREAD

**Discussion by ijinesh** [sebanc/brunch#1332] [General]
Hi,
A basic question on the kernel version used in Brunch r94 stable 20211121. And how can I change kernel version to suit my requirement.

Also, I am not able to saving changes in Brunch config (using Brunch r94 stable 20211121). It says Brunch config is saved and restarts the system. However, when it restarts changes are not reflected in the config.
I am using Brunch r94 stable 20211121 on Volteer image on Surface Pro 7.
Thanks in advance.

**Comment by ycavan** [comment]
re: config

something you could try would be to mount partition 12 and edit the efi/boot/settings.cfg file manually.

**Reply by ijinesh** [reply]
Thanks @ycavan 

I made changes in Grub2win itself and it works.
